lib/index.js

```diff
@@ -76,15 +78,19 @@ function printResults(results, c) {
     }).join("\n");
 }
 
-function searchVOD(vodID, c) {
+function searchVOD(vodID, args) {
```

Comment:
I'd rather have three arguments here than a magic args object. If there will be more arguments added it will most likely result in a major version jump either way.